apps/web/src/mocks/mockServer.ts

```diff
@@ -0,0 +1,11 @@
+import { rest } from 'msw';
+import { setupServer } from 'msw/lib/node';
+import { metaData } from '../components/TrendingItemSuggestion/TrendingItemSuggestion.utils';
+
+const server = setupServer(
```

Comment:
I spun up the `TrendingSuggestions` component in the app and it's working really well, fantastic! I noticed there is some possible overlap between this file and the `handlers.ts` file. Do they both do different things? I'm just not sure how they both work, so thought I'd ask.

Reply:
After going back and looking at `mockServer` & `browser` I thought the same thing. I tried re-configuring the `TrendingSuggestions.test.tsx` file so that it would depend on the browser request, but couldn't get it to work. After a little research it looks like you need to create a separate server for your test cases, and I created the separate file so we didn't have to define a new server in every component that depends on a MSW response.